_posts/2018-07-17-its-a-lot-more-snappy.md

```diff
@@ -166,6 +137,6 @@ worker.onmessage = async (event: MessageEvent): Promise<void> => {
 };
 ```
 
-Adopting this approach meant that we offload the computationally expensive task of writing all this data to IndexedDB. The user can now continue on the main thread and the app will remain responsive without any noticeable decrease in responsiveness over time. Having all this data stored in IndexedDB also means, offline usage is now possible and subsequent loads of the dataset are no longer a concern at all because the data can be accessed instantly from the local store.
+Profiling the app before and after the changes revealed that using a background process increased the max heap size by 20mb. Even though memory usage increased, the benefit of implementing the web worker was that we could now offload the computationally expensive task of writing all this data to IndexedDB. The user can now continue on the main thread and the app will remain responsive without any noticeable decrease in responsiveness over time.
 
-The Chrome dev tools also reveal that using a background process does not result in a significant increase in memory usage. Having noticed these improvements, our PM said “After testing the app, it’s now a lot more snappy!”
+Using PWA technology enabled us to deliver a solution that would have otherwise not been possible with a conventional web app. Storing this data in IndexedDB means, offline usage is now possible and subsequent loads of the dataset are no longer a concern at all because the data can be accessed instantly from the local store. Having noticed these improvements, our client said “After testing the app, it’s now a lot more snappy!”
```

Comment:
👍 

Really good post.  

If you compare to the original draft, the post is now much more engaging and really highlights the message you're trying to convey.  This isn't a criticism - just a natural part of the authoring process.